core-data/resolvers.js

```diff
@@ -7,15 +7,29 @@ import apiRequest from '@wordpress/api-request';
  * Internal dependencies
  */
 import { setRequested, receiveTerms, receiveMedia } from './actions';
+import { getTaxonomyRESTBase } from './settings';
+
+/**
+ * Requests terms from the REST API, yielding action objects on request
+ * progress.
+ *
+ * @param {string} taxonomy Taxonomy for which terms should be requested.
+ */
+export async function* getTerms( taxonomy ) {
+	yield setRequested( 'terms', taxonomy );
+	const restBase = getTaxonomyRESTBase( taxonomy );
```

Comment:
I wonder if we could avoid the `apiRequest.settings` by fetching the rest base from `/wp/v2/taxonomies` or something? A resolver calling another resolver? Any thoughts on this. 

Just trying to avoid as much globals as we can

Reply:
Thinking more about this, `getTaxonomy` could be a selector with a side effect. 

`const { rest_base } = yield select( 'core' ).getTaxonomy( taxonomy )` which will call the side effect only once.

Reply:
I guess we need a `resolve` here because we want to wait for fulfillment

Reply:
That might work. We'd definitely want to bootstrap this data, to avoid the additional request occurring in the client before being able to resolve terms. Bootstrapping support might be something we want anyways.

Part of the goal in assigning `wpApiSettings` into `apiRequest` was to avoid the introduction of another global, since `apiRequest` is already faked into module form as `@wordpress/api-request`.

Reply:
`wp.apiRequest.settings` is definitely better than `wpApiSettings` or `wp.api.*`, we'd have a unique root global dependency `wp.apiRequest`. If you think we can try the resolver approach separately, I'm fine with that.